benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -927,20 +932,22 @@ void GraphicsBenchmarkApp::DrawExtraInfo()
     ImGui::Text("%d x %d", width, height);
     ImGui::NextColumn();
 
-    const uint32_t quad_count    = pFullscreenQuadsCount->GetValue();
-    const float    dataWriteInGb = (static_cast<float>(width) * static_cast<float>(height) * 4.f * quad_count) / (1024.f * 1024.f * 1024.f);
-    ImGui::Columns(2);
-    ImGui::Text("Write Data");
-    ImGui::NextColumn();
-    ImGui::Text("%.2f GB", dataWriteInGb);
-    ImGui::NextColumn();
-
-    const float bandwidth = dataWriteInGb / gpuWorkDurationInSec;
-    ImGui::Columns(2);
-    ImGui::Text("Write Bandwidth");
-    ImGui::NextColumn();
-    ImGui::Text("%.2f GB/s", bandwidth);
-    ImGui::NextColumn();
+    const uint32_t quad_count = pFullscreenQuadsCount->GetValue();
+    if (quad_count) {
+        const float dataWriteInGb = (static_cast<float>(width) * static_cast<float>(height) * 4.f * quad_count) / (1024.f * 1024.f * 1024.f);
+        ImGui::Columns(2);
```

Comment:
I think actually within this function it only needs a `ImGui::Columns(2)` at the beginning and an `ImGui::Columns(1)` at the end (so that the knobs are not rendered in the left column of a 2 column section). All the other calls to `ImGui::Columns(2);` are redundant

Reply:
changed as suggested